front/gatsby/src/pages/onboarding.tsx

```diff
@@ -1,45 +1,50 @@
-// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// Copyright 2018-2019 @paritytech/nomidot authors & contributors
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { RouteComponentProps } from '@reach/router';
 import { AccountsContext } from '@substrate/context/src';
-import {
-  AccountsList,
-  Breadcrumbs,
-  Circle,
-  Modal,
-  polkadotOfficialTheme,
-  Transition,
-} from '@substrate/ui-components/src';
-import React, { useContext } from 'react';
+import { AccountsList, Modal, Transition } from '@substrate/ui-components/src';
+import React, { useContext, useEffect, useState } from 'react';
 
-const ONBOARDING_STEPS = ['stash', 'controller'];
-
-// eslint-disable-next-line
-export const Onboarding = (props: RouteComponentProps): React.ReactElement => {
+const Onboarding = (): React.ReactElement => {
   const { injectedAccounts } = useContext(AccountsContext);
-  // const [stash, setStash] = useState();
-  // const [controller, setController] = useState();
+  const [isComponentMounted, setIsComponentMounted] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsComponentMounted(true);
+
+    return (): void => setIsComponentMounted(false);
+  }, []);
+
+  // TODO: save to DB not local storage
+  const handleSelectAccount = ({
+    currentTarget: {
+      dataset: { address },
+    },
+  }: React.MouseEvent<HTMLElement>): void => {
+    if (address) {
+      localStorage.setItem('exploringAs', address);
```

Comment:
Could you add a `window.alert('TODO')` or something? While playing I wasn't sure if it was a bug or not implemented.